pkg/apiserver/manager/hwameistor/localstoragenode_controller.go

```diff
@@ -596,25 +596,22 @@ func (lsnController *LocalStorageNodeController) StorageNodePoolDiskGet(page hwa
 	nodeKey := client.ObjectKey{
 		Name: page.NodeName,
 	}
-
+	// get localdisk information
+	devicePath := hwameistorapi.DEV + page.DiskName
+	localDisks, e := lsnController.ldHandler.ListLocalDiskByNodeDevicePath(page.NodeName, devicePath)
+	if e != nil {
+		log.WithError(e).Errorf("Failed to get localdisk,nodeName:%v,devPath:%v", page.NodeName, devicePath)
+		return nil, e
+	}
+	localDisk := &localDisks[0]
```

Comment:
may panic here when filtered localDisks are empty

Reply:
ok,I try to modify it